src/main/java/net/snowflake/client/jdbc/SFAsyncResultSet.java

```diff
@@ -23,7 +18,7 @@
 /** SFAsyncResultSet implementation */
 class SFAsyncResultSet extends SnowflakeBaseResultSet implements SnowflakeResultSet, ResultSet {
   private final SFBaseResultSet sfBaseResultSet;
-  private ResultSet resultSetForNext = new EmptyResultSet();
+  private ResultSet resultSetForNext = new SnowflakeResultSetV1.EmptyResultSet();
```

Comment:
No need for duplicate EmptyResultSet class; can be imported from SnowflakeResultSetV1. This empty result class should never be used in SFAsyncResultSet anyway if the user is calling functions correctly.